pkg/services/ngalert/api/api_ruler.go

```diff
@@ -187,6 +191,11 @@ func (srv RulerSrv) RoutePostNameRulesConfig(c *models.ReqContext, ruleGroupConf
 	// TODO check quota
 	// TODO validate UID uniqueness in the payload
 
+	//TODO: Should this belong in alerting-api?
+	if ruleGroupConfig.Name == "" {
```

Comment:
This validates that rule groups must have a name. Is his correct?

Reply:
We have an endpoint that allows users to delete rules by rule group and namespace - without this becomes unusable.